frame/support/src/traits/tokens/fungibles.rs

```diff
@@ -281,9 +275,41 @@ pub trait Destroy<AccountId>: Inspect<AccountId> {
 	///
 	/// If successful, this function will return the actual witness data from the destroyed asset.
 	/// This may be different than the witness data provided, and can be used to refund weight.
-	fn destroy(
-		id: Self::AssetId,
-		witness: Self::DestroyWitness,
-		maybe_check_owner: Option<AccountId>,
-	) -> Result<Self::DestroyWitness, DispatchError>;
+	fn start_destroy(id: Self::AssetId, maybe_check_owner: Option<AccountId>) -> DispatchResult;
+
+	/// Destroy all accounts associated with a given asset.
+	/// `destroy_accounts` should only be called after `start_destroy` has been called, and the
+	/// asset is in a `Destroying` state
+	///
+	/// Due to weight restrictions, this function may need to be called multiple
+	/// times to fully destroy all accounts. It will destroy `RemoveItemsLimit` accounts at a
```

Comment:
there is no `RemoveItemsLimit` in this context

Reply:
Yeah, I've updated the docs. The max_items value does the job in this case. Thanks for spotting it.